pkg/util/network/network.go

```diff
@@ -54,6 +55,21 @@ func CreateSriovPolicy(clientSet *testclient.ClientSet, generatedName string, op
 	return err
 }
 
+func GetSriovNics(pod *k8sv1.Pod, ifcPrefix string) ([]string, error) {
```

Comment:
Can you rename this to "GetNicsByPrefix"?
This does not make any logic about those interfaces being sriov

Reply:
done